docs/reference/deploy/master-config-reference.rst

```diff
@@ -1747,7 +1747,7 @@ Determines if users should be automatically created in Determined upon successfu
 ``groups_attribute_name``
 =========================
 
-The claim name that specifies group memberships in SAML.
+The name of the attribute passed in through the claim that specifies group memberships in SAML.
```

Comment:
Could you change this to "of the attribute passed in through the SAML Assertion"

Reply:
done